spec/subtitle_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Subber::Subtitle do
+  let(:attributes) { raise NotImplementedError }
+  subject { described_class.new(attributes) }
+
+  describe 'as_json' do
+    let(:attributes) do
+      {
+        counter: 1,
+        start_time: 2000,
+        end_time: 5000
+      }
+    end
+
+    it 'returns the correct hash' do
+      expect(subject.as_json)
+        .to eql(
+          {
+            'counter' => 1,
+            'start_time' => 2000,
+            'end_time' => 5000,
+            'content' => nil
+          }
+        )
+    end
+  end
+end
```

Comment:
This looks ok but I have a few suggestions regarding this spec. Check the following snippet:

```rb
describe Subber::Subtitle do
  let(:attributes) { raise NotImplementedError }
  let(:subtitle) { described_class.new(attributes) }

  describe '#as_json' do
    let(:attributes) { ... }
    let(:expected_result) { ... }

    subject { subtitle.as_json }
    it { is_expected.to eql(expected_result) }
  end
end
```